.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
   - npm install -g codecov
   - npm install
```

Comment:
```suggestion
  - npm ci
```

Reply:
Discussion in https://github.com/jupyterhub/configurable-http-proxy/pull/228#issuecomment-598153327